src/pages/experiments/[experimentId]/plots-and-tables/embedding-continuous/index.jsx

```diff
@@ -80,30 +80,27 @@ const EmbeddingContinuousIndex = ({ experimentId }) => {
     updatePlotWithChanges({ shownGene: geneName });
   };
 
-  if (!config) {
-    return (
-      <center>
-        <Loader experimentId={experimentId} />
-      </center>
-    );
-  }
-
   const renderExtraPanels = () => (
     <>
       <Panel header='Gene Selection' key='666'>
-        <Search
-          style={{ width: '100%' }}
-          enterButton='Search'
-          defaultValue={config.shownGene}
-          onSearch={(val) => changeDislayedGene(val)}
-        />
+        {config ? (
+          <Search
+            style={{ width: '100%' }}
+            enterButton='Search'
+            defaultValue={config.shownGene}
+            onSearch={(val) => changeDislayedGene(val)}
+          />
+        ) : <Skeleton.Input style={{ width: 200 }} active />}
```

Comment:
Similarly here. Instead of delaying loading the page altogether, just delay loading the component that depends on `config`.